README.md

```diff
@@ -1 +1,73 @@
-# equalizer
\ No newline at end of file
+# Wolox Equalizer
+[![circle-ci](https://img.shields.io/circleci/project/github/Wolox/wolox-equalizer.svg)](https://circleci.com/gh/Wolox/wolox-equalizer)
+[![npm](https://img.shields.io/npm/v/wolox-equalizer.svg)](https://www.npmjs.com/package/wolox-equalizer)
+
+[![FEArmy](./assets/FEA_icon.png)](https://github.com/orgs/Wolox/teams/front-end-army/members)
```

Comment:
Check this logo, I think it's not the correct link path

Reply:
This link (https://github.com/orgs/Wolox/teams/front-end-army/members) works.

Reply:
Link is fine but the FEA logo is not in that path

Reply:
it's the certified icon, I saw it and it's ok